packages/teams-js/src/public/video.ts

```diff
@@ -216,4 +216,92 @@ export namespace video {
   export function isSupported(): boolean {
     return ensureInitialized(runtime) && runtime.supports.video ? true : false;
   }
+
+  /**
+   * Namespace to get video frames from a media stream
```

Comment:
Please expand this comment to explain why a developer would want to use mediaStream instead of the functions in the parent capability